server/test/trips.test.js

```diff
@@ -169,13 +147,64 @@ describe('Trip Tests', () => {
 			});
 	});
 
+	// TEST FOR CANNOT GET ALL TRIPS
+	it('GET/api/v1/trips Should not fetch all trips', (done) => {
+		chai
+			.request(app)
+			.get('/api/v1/trips')
+			.set('authorization', `Bearer ${userToken}`)
+			.end((err, res) => {
+				console.log(res.body);
+				res.should.have.status(404);
+				res.should.should.be.a('object');
+				done();
+			});
+	});
+
+	// TEST FOR CANNOT GET A SPECIFIC TRIP TEST
+	it('GET/api/v1/trips/:trip-id Should fetch a specific trip', (done) => {
+		chai
+			.request(app)
+			.get(`/api/v1/trips/${1}`)
+			.set('authorization', `Bearer ${userToken}`)
+			.end((err, res) => {
+				res.should.have.status(404);
+				res.should.should.be.a('object');
+				done();
+			});
+	});
+
+	// TEST FOR CREATE A NEW TRIP 
+	it('POST/api/v1/trips Should create a new trip', (done) => {
+		const trip = {
+			seating_capacity: '23',
+			bus_license_number: 'KZE 432',
+			origin: 'Nairobijhjjghghghbhjbj',
+			destination: 'KIgalijhjjjhug',
+			trip_date: '21/09/2019',
+			fare: '4000',
```

Comment:
Fare isn't a string

Reply:
My validation for the fare I used regex which takes in strings.